.travis.yml

```diff
@@ -59,6 +59,10 @@ matrix:
   - name: "Clang Release"
     compiler: clang
     config: Release
+  - name: "AArch64 GCC Release"
+    arch: arm64
+    compiler: gcc
+    config: Release
```

Comment:
I suggest moving this to line 44, so that it follows the similar "GCC Debug" and "GCC Release" immediately.

Reply:
Done.